nexus/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #![feature(async_closure)]
+#![feature(int_roundings)] // used in crate::cidata
```

Comment:
Nice PR!

Out of curiosity, how essential is this?  I think we're trying to get Omicron to stable Rust but it looks like this option is still unstable.

Reply:
It's not terribly essential, since I can just copy its implementation. Should I?

Reply:
We're currently able to build with stable on Linux and illumos, but not macOS. That'll be gated by the `asm_sym` feature, required for the internals of the `usdt` crate. That's on the path to stabilization, but it'll likely be at least a few more months. I agree that we should try to avoid new features that pin us to nightly though.